rfcs/0000-error-handling.md

```diff
@@ -0,0 +1,153 @@
+# RFC-0000: Subgraph error handling and reporting
+
+<dl>
+  <dt>Author</dt>
+  <dd>Leonardo Y. Schwarzstein</dd>
+    <dt>RFC pull request</dt>
+  <dd>URL</dd>
```

Comment:
Don't forget to add https://github.com/graphprotocol/rfcs/pull/18/files in place of `URL`.